coverage.py

```diff
@@ -68,8 +68,12 @@ def __init__(self, _db, batch_size=10, cutoff_time=None,
                  uploader=None, providers=None, **kwargs):
         output_source, made_new = get_one_or_create(
             _db, DataSource,
-            name=DataSource.INTERNAL_PROCESSING, offers_licenses=True,
+            name=DataSource.INTERNAL_PROCESSING
```

Comment:
By default, `offers_licenses` is set to False when this DataSource is created in core. This was attempting to create a second INTERNAL_PROCESSING data source, which caused a database integrity error. Since we always want offers_licenses to be True, I just changed it to set `offers_licenses` after fetching the `DataSource` object.